packages/shared/src/components/post/PostHeaderActions.tsx

```diff
@@ -96,13 +97,15 @@ export function PostHeaderActions({
       }
     }
   };
+  const readButtonText =
+    post.type === PostType.VideoYouTube ? 'Watch video' : 'Read post';
```

Comment:
Seen two cases of this already, perhaps also a good util function.
Let's say we want to adjust the copy, would now need to happen in multiple places.

Reply:
Yes exactly, just so it's managed from 1 spot :)

Reply:
Refactored in a7ac75a, now just one place to update it

Reply:
Nice love it 🍡